cluster/local/kind.sh

```diff
@@ -49,7 +49,7 @@ case "${1:-}" in
     ;;
   update)
     helm_tag="$(cat _output/version)"
-    copy_image_to_cluster ${BUILD_IMAGE} "${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}" "${KIND_NAME}"
+    copy_image_to_cluster ${BUILD_IMAGE} "xpkg.upbound.io/${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}" "${KIND_NAME}"
```

Comment:
instead of doing the same change 3 times, would it be simpler/less effort if we continue copying the image as is, using `${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}"` as the image ref, and then when interacting with to pass the arg:

```
--set image.repository=${PROJECT_NAME}
```

Orthogonal to that, we could extract `"${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}"` into a variable and use it everywhere, so that future changes would be one-liner?

Same goes for default helm cli args.

Reply:
Refactored to use a variable with updated value.